cpp/src/io/orc/aggregate_orc_metadata.cpp

```diff
@@ -14,7 +14,9 @@
  * limitations under the License.
  */
 
-#include "aggregate_orc_metadata.hpp"
+#include <io/orc/aggregate_orc_metadata.hpp>
```

Comment:
We prefer quoted includes when it's in the same directory (the way it was before), right?